amy/dashboard/tests/test_landing_page.py

```diff
@@ -67,7 +74,29 @@ def test_no_inactive_events(self):
         self.assertNotIn(unresponsive, response.context['unpublished_events'])
         self.assertNotIn(cancelled, response.context['unpublished_events'])
         self.assertNotIn(completed, response.context['unpublished_events'])
+    
+    def test_default_events(self):
+        """The admin dashboard shows the events assigned to the current logged in 
+        user by default."""
+        upcoming_event = Event.objects.upcoming_events().first()
+        upcoming_event.assigned_to = self.admin
+        upcoming_event.save()
+
+        unpublished_event = Event.objects.active().unpublished_events().first()
+        unpublished_event.assigned_to = self.admin
+        unpublished_event.save()
+
+        
```

Comment:
Two blank lines inside function is probably a PEP8 style violation. Do you have pre-commit and black installed?

If not, run:

```
$ pip install -r dev_requirements.txt
$ pre-commit install
```

This will make sure to run Black upon commit.

We'll soon add Flake8 to the "stack" as well.

Reply:
I did miss that step. Thanks!